app/views/users/settings/account/preferences/index.html.erb

```diff
@@ -133,6 +133,27 @@
               </div>
             </div>
           </div>
+          <br>
```

Comment:
I would move this out of the `notifications` segment - one level up, to be on the same level as notifications & time zone containers.